include/deal.II/lac/sparsity_pattern.h

```diff
@@ -306,15 +306,16 @@ namespace SparsityPatternIterators
 
 
 /**
- * A class that can store which elements of a matrix are nonzero (or, in fact,
- * <i>may</i> be nonzero) and for which we have to allocate memory to store
- * their values. This class is an example of the "static" type of sparsity
+ * A class that can store which elements of a matrix are nonzero (or, to be
+ * precise, <i>may</i> be nonzero) and for which we have to allocate memory to
+ * store their values. This class is an example of the "static" type of sparsity
  * patters (see
```

Comment:
```suggestion
 * patterns (see
```
can you fix it while we are here?